libr/parse/p/parse_att2intel.c

```diff
@@ -42,8 +42,8 @@ static int replace(int argc, const char *argv[], char *newstr) {
 					if (ops[i].str[j]>='0' && ops[i].str[j]<='9') {
 						const char *w = argv[ ops[i].str[j]-'0' ];
```

Comment:
proper spacing around - and <= operators across the whole file, not just the lines you changed there.